src/System.CommandLine.Benchmarks/DragonFruit/Perf_CommandLine_EntryPoint.cs

```diff
@@ -0,0 +1,165 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.CommandLine.Benchmarks.Helpers;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using BenchmarkDotNet.Attributes;
+
+namespace System.CommandLine.Benchmarks.DragonFruit
+{
+    /// <summary>
+    /// This is an end-to-end benchmark that measures the performance of
+    /// <see cref="System.CommandLine.DragonFruit.CommandLine.ExecuteAssemblyAsync"/>
+    /// including the execution of Main method per se.
+    /// </summary>
+    [BenchmarkCategory(Categories.DragonFruit)]
+    [InvocationCount(3000)]
```

Comment:
why do we need to specify this here? how is the harness doing a poor job picking an iteration count?

Reply:
oh, I see your BDN bug now.

Reply:
1. *why do we need to specify this here? how is the harness doing a poor job picking an iteration count?*

Yes, the automatically selected values for iteration count were too small. BND was persistently giving me `MinIterationTime` warnings which are a serious problem, so I had to manually increase the invocations count to the value to the point when I stopped getting those warnings. I hope this value will also be enough for other machines where people will be running those tests.
```
// * Warnings *
MinIterationTime
  Perf_CommandLine_EntryPoint.'ExecuteAssemblyAsync entry point search.': IterationTime=250.0000 ms, MaxIterationCount=20, MinIterationCount=15, WarmupCount=1 -> The minimum observed iteration time is 922.8077 us which is very small. It's recommended to increase it.
```

2. *oh, I see your BDN bug now.*
No. It's a separate issue.

Reply:
understood.